mojo/stdlib/stdlib/collections/list.mojo

```diff
@@ -234,21 +234,34 @@ struct List[T: ExplicitlyCopyable & Movable, hint_trivial_type: Bool = False](
            Defaults to `False`.
     """
 
+    # Aliases
+    alias _has_trivial_del = Bool(T.__del__is_trivial)
+    alias _has_trivial_copy = hint_trivial_type  # Bool(T.__copy_is_trivial)
+    alias _has_trivial_move = Bool(T.__moveinit__is_trivial)
+    alias _is_fully_trivial = (
+        Self._has_trivial_move
+        and Self._has_trivial_copy
+        and Self._has_trivial_del
+    )
+
+    alias _Span = Span[T, MutableAnyOrigin]
+    alias _U = Self._Span.UnsafePointerType
```

Comment:
I usually try not to block on naming but I'm pretty against single letter aliases like this